specification/recoveryservicessiterecovery/resource-manager/readme.md

```diff
@@ -72,6 +72,15 @@ directive:
 ```
 
 
+### Tag: package-2024-10
+
+These settings apply only when `--tag=package-2024-10` is specified on the command line.
+
+```yaml $(tag) == 'package-2024-10'
+input-file:
+  - Microsoft.RecoveryServices/stable/2024-10-01/service.json
```

Comment:
All the `"x-ms-parameter-location": "client"` in service.json shall be removed otherwise it will cause unexpected breakings for SDK.

Reply:
Removed "x-ms-parameter-location": "client"

Reply:
I already added it in readme.python.md so you don't need remove it for python now.